plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/extension/MetalavaExtension.kt

```diff
@@ -98,4 +98,26 @@ open class MetalavaExtension {
      * The default is "current".
      */
     var releaseType = "current"
+
+    /**
+     * The directories to search for source files. An exception will be thrown is the named
+     * directories are not direct children of the project root. The default is "src".
+     *
+     * @see ignoreSourcePaths
+     */
+    var sourcePaths = mutableSetOf("src")
+
+    /**
+     * The sub-directories of each directory specified by [sourcePaths] that should not be searched
+     * for source files. Directory name matching is conducted using case-insensitive equality. The
+     * default values are `("test", "commonTest", "androidTest", "iosTest", "jsTest")`.
+     *
+     * @see sourcePaths
+     */
+    var ignoreSourcePaths = mutableSetOf("test", "commonTest", "androidTest", "iosTest", "jsTest")
```

Comment:
I'm assuming these are the dirnames you wanted?

Reply:
Yea these are a good baseline! Of course multiplatform supports a bunch of other targets, but these are probably the most popular ones.